packages/ui-app/src/types.d.ts

```diff
@@ -25,3 +25,8 @@ export type BaseContext = {
     }
   }
 };
+
+export type UnlockI18n = {
```

Comment:
Remove if we are not passing these through - as it should be React.ReactNode going in

Reply:
Existing code prior to this PR uses `UnlockI18n` in ui-signer and app-toolbox, I've left that implementation unchanged... or should I change that to `React.ReactNode` too?